DEVELOPMENT.md

```diff
@@ -2,15 +2,41 @@
 
 ## Prerequisites
 
-In order to build Eunomia, you'll need the following:
+Apart from setting up access to a local Kubernetes cluster (described in the [Prerequisites](./GETTING_STARTED.md#Prerequisites) section in the Getting started guide), you will need the following in order to build and test Eunomia:
 
 - [Git](https://git-scm.com/downloads)
 - [Go 1.12+](https://golang.org/dl/)
 - [Docker](https://docs.docker.com/install/)
 - [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
-- Access to a Kubernetes cluster
-  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (optional)
-  - [Minishift](https://www.okd.io/minishift/) (optional)
+- [Minishift](https://github.com/minishift/minishift/releases) (optional; a local OpenShift cluster)
```

Comment:
What would you think about keeping the list as it was before (still mentioning minikube), but adding a link to Getting Started to follow for detailed guide? Also because IIUC, if using minishift, user wouldn't actually need minikube and the other stuff from Prerequisites, no?

Reply:
I'll rework it in such a way that all instructions will be in one place and I'll be only referencing them from everywhere

Reply:
I'm ok with some information being repeated/duplicated, I believe it can sometimes help readability. I see DEVELOPMENT.md vs. GETTING_STARTED.md as having very different target audience and goal. For example, I could imagine thinking of DEVELOPMENT as having comprehensive & exhaustive "RTFM-grade" info, while GETTING_STARTED being a single, focused, "simple happy path" tutorial/story, with some aspects repeated from DEVELOPMENT.